### PR TITLE
Update Ghost Button layout

### PR DESCRIPTION
**Pull request by vladmoroz**
Updated Ghost Button layout so we can use it with 25 px and 15 px icons alike:

Screenshot 2020-03-25 at 10 01 25

No need for a new size variant this way. Closes https://github.com/modulz/radix/issues/192

### CODE REVIEW

**Comment by vladmoroz** [comment]
@peduarte Yep, I'm aware about the variant approach. I think it makes sense when you need text in the ghost button.

But we only use ghost buttons with icons, so paddings suffice for consistent sizes. If we ever need to use ghost buttons with text, we should dig deeper to re-evaluate its design, because the roundings are too strong for that. I don't think we need to handle theoretical cases that we shouldn't have in the first place right now.

Does this make sense or are you still convinced it has to be a size variant?

**Comment by peduarte** [comment]
@vladmoroz I understand and agree, my main concern is that we do sometimes use `GhostButton` with text, and the CSS has a bunch of text related properties, which implies it can be used with text only children.

image

Reading the code as is, I'm confused by how the usage is. 

It's no big deal, specially since soon this will all be irrelevant, but I think using a variant is more inline with the rest of the Components API.

**Comment by vladmoroz** [comment]
OK, I added a size variant.
Let's take a look at our ghost button usage separately then.

File: packages/radix/src/components/GhostButton.tsx [hunk 1]
```diff
@@ -55,6 +54,26 @@ export const GhostButton = React.forwardRef<HTMLButtonElement, GhostButtonProps>
...
+                },
+              },
+            },
+            1: {
+              button: {
+                normal: {
+                  height: theme.sizes[6],
+                  minWidth: theme.sizes[6],
+                  paddingTop: theme.space[1],
+                  paddingBottom: theme.space[1],
```
Comment (peduarte) [inline packages/radix/src/components/GhostButton.tsx]: Is the font size not being increased on purpose?

Comment (vladmoroz) [reply packages/radix/src/components/GhostButton.tsx]: Adding 👌

**Comment by vladmoroz** [comment]
Closing in favor of https://github.com/modulz/radix/pull/238

**Pull request closed without merging**